### PR TITLE
Update chat extension API for cache

### PR DESCRIPTION
**Pull request by Steven-Chan**
connect #113

### CODE REVIEW

File: SKYKitChat/Classes/Core/SKYChatExtension.m [hunk 1]
```diff
@@ -279,13 +282,23 @@ - (void)fetchConversationWithConversationID:(NSString *)conversationId
 - (void)fetchMessagesWithIDs:(NSArray<NSString *> *)messageIDs
                   completion:(SKYChatFetchMessagesListCompletion)completion
 {
+    if (completion) {
+        [self.cacheController
+            fetchMessagesWithIDs:messageIDs
+                      completion:^(NSArray<SKYMessage *> *_Nullable messageList,
+                                   NSArray<SKYMessage *> *_Nullable deletedMessageList,
+                                   BOOL isCached, NSError *_Nullable error){
+                          // TODO: call completion handler of the api
```
Comment (rickmak) [inline SKYKitChat/Classes/Core/SKYChatExtension.m]: I think you missed to update the callback at 379e336.

**Review by rickmak — CHANGES_REQUESTED** [review]
Just one comment, it looks good.

**Comment by Steven-Chan** [comment]
@rickmak updated

**Comment by Steven-Chan** [comment]
@rickmak so sorry, lint failed, fixed

**Pull request merged**